### DISCUSSION TITLE
Octernship Related

### DISCUSSION THREAD

**Discussion by newsid2024** [community/community#50290] [GitHub Education]
Its showing 12 projects available but u can still apply to 4 but i am not able to ? What does it mean ?
<img width="1440" alt="Screenshot 2023-03-17 at 11 03 04 AM" src="https://user-images.githubusercontent.com/108874631/225821079-4d030878-8081-4de4-b1e3-de92c81747f4.png">

**Comment by naimur-reza** [comment]
In the context of Octernship on GitHub, it's possible that the 12 projects mentioned are the total number of projects that were available for the program, but only 4 of them are still open for applications.

This means that there were initially 12 projects that you could have applied for during the application period, but 8 of them may have already been filled or closed for new applications. Therefore, you are only able to apply for the remaining 4 projects that are still open for new applications.

It's important to keep in mind that these details are specific to Octernship on GitHub and may not apply to other programs or situations. If you have any specific questions about the Octernship program or your application, it may be best to reach out to the program organizers directly for clarification.

**Comment by Abhishek-569** [comment]
Click on `I'm Interested` link below the projects to apply to them. But if You are a verified student and hence can see the Octernship opportunities, you are unable to apply to them due to one or more following reasons:

    You are not an active contributor on GitHub

>  ✨ Tip: It is a good practice to contribute to open source regularly and [renew](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/apply-to-github-global-campus-as-a-student#expiration-and-renewals) your student status if it expires.

**Comment by rameshchandra8520** [comment]
It means there are 12 Projects available, and you can apply to any 4 of them. And you can only apply to a maximum of four projects per year.

**Comment by saaxena** [comment]
it means that you can apply to  a maximum  number of 4 projects throughout the year
